### PR TITLE
US103573 - Adding HM wrapper for activity icon

### PR DESCRIPTION
**Pull request by mtjandra**
* HM wrapper for draft icon

### CODE REVIEW

**Comment by mtjandra** [comment]
There seems to be some risk doing it this way. So for the time being I am going to get the evaluation from the unassessed activity instead of breaking this into its own component. https://git.dev.d2l/projects/CORE/repos/le/pull-requests/14288/overview?commentId=335563

File: components/d2l-activity-evaluation-icon/d2l-activity-evaluation-icon.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import 'd2l-polymer-siren-behaviors/store/entity-behavior.js';
+import 'd2l-icons/d2l-icon.js';
```
Comment (svanherk) [inline components/d2l-activity-evaluation-icon/d2l-activity-evaluation-icon.js]: You don't need to import the icons stuff here

File: demo/d2l-activity-evaluation-icon/d2l-activity-evaluation-icon.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+		<title>d2l-evaluation-hub demo</title>
```
Comment (svanherk) [inline demo/d2l-activity-evaluation-icon/d2l-activity-evaluation-icon.html]: minor: `d2l-activity-evaluation-icon demo`

File: test/d2l-activity-evaluation-icon/d2l-activity-evaluation-icon.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		test('base configured as draft when data indicates evaluation in draft state', function(done) {
+			loadPromise('data/activity-draft.json', evaluationIcon)
+				.then(function(configuredEvaluationIcon) {
+					var baseIcon = configuredEvaluationIcon.shadowRoot.querySelector('d2l-activity-evaluation-icon-base');
+					assert.equal(true, baseIcon.hasAttribute('draft'));
+					done();
+				});
+		});
+
+		test('base no configured as draft when data does not contani evaluation information', function(done) {
```
Comment (svanherk) [inline test/d2l-activity-evaluation-icon/d2l-activity-evaluation-icon.js]: minor: "not", "contain"

**Pull request merged**